src/core/elrs.c

```diff
@@ -364,4 +368,43 @@ void msp_ht_update(uint16_t pan, uint16_t tilt, uint16_t roll) {
 
 bool elrs_headtracking_enabled() {
     return headtracking_enabled;
+}
+
+void elrs_clear_osd() {
+    for (int i = 0; i < HD_VMAX; i++) {
```

Comment:
maybe `memset` is more appropriate here.

Reply:
Can't use memset because the underlying type is uint16_t

Reply:
right, missed that